packages/synapse-react-client/src/components/JsonSchemaForm/templates/ArrayFieldTemplate.tsx

```diff
@@ -58,12 +58,15 @@ function ArrayFieldTemplate<
    */
   useEffect(() => {
     if (
+      // Do not manipulate if the widget is hidden (e.g. 'scopeIds' in an EntityView)
+      // FIXME: We should probably remove this hook and migrate to a design that accommodates empty arrays.
+      uiOptions.widget !== 'hidden' &&
```

Comment:
Fixes issue where if you had an entity with an empty array property (in one case, an EntityView with an empty scope), a single undefined value would be added to the array, which would be invalid.

Added a note to remove this--the easiest thing would be to allow empty arrays in the UI.